torch/distributed/_sharded_tensor/ops/__init__.py

```diff
@@ -0,0 +1 @@
+from .linear import sharded_linear
```

Comment:
nit: for the dir structure. should sharded_linear be under _sharded_tensor/modules or /ops ?  Probably it's closer to modules